database/arango/arango.js

```diff
@@ -0,0 +1,78 @@
+require("dotenv").config();
+const { Database } = require("arangojs");
+const { sendFetch } = require("../../lib/sendFetch");
+const logger = require("./../../lib/log")(__filename);
+
+const arangoModule = {};
+let db;
+
+arangoModule.checkIfDatabaseExists = async (name) => {
+  // returns array of objects containing a '_name' key.
+  // Ex [{ _name: 'lol' }, { _name: 'cakes' }]
+  const names = await db.databases();
+  return names.map((db) => db._name).includes(name);
+};
+
+arangoModule.startArangoDB = async () => {
+  db = new Database({
+    url: process.env.ARANGO_URL,
+    databaseName: "_system",
+    auth: {
+      username: process.env.ARANGO_USER,
+      password: process.env.ARANGO_PW,
+    },
+  });
+};
+
+arangoModule.closeArangoDB = () => db.close();
+
+arangoModule.createAccount = async (account) => {
+  if (!account) return;
+  const { username, dbPassword } = account;
+  if (!username || !dbPassword) return;
+  if (await arangoModule.checkIfDatabaseExists(username)) return;
+
+  try {
+    await db.createDatabase(username, {
+      users: [{ username, password: dbPassword }],
+    });
```

Comment:
How about putting `log.info("Created ArangoDB account")`? Not only this one but also other functions

Reply:
Will do. Thanks!